app/collins/controllers/actions/ipaddress/GetPoolsAction.scala

```diff
@@ -38,14 +39,25 @@ case class GetPoolsAction(
           } else {
             IpAddresses.getPoolsInUse()
           }
-        format(pools)
+        val ipmiPools: Set[String] =
+          if (all) {
+            IpmiInfo.AddressConfig.map(_.poolNames).getOrElse(Set())
```

Comment:
Check that this handles mapping the default pool to the proper name correctly? I.e. if you have a config that uses an implicit network (not using the pools = {...} syntax)

Also, does this stuff need to be reverted now that you have a separate endpoint for ipmi pools?